packs/actions/supabase/register.ts

```diff
@@ -0,0 +1,153 @@
+import type { AppContext } from "$store/apps/site.ts";
+import { Secret } from "apps/website/loaders/secret.ts";
+import { createClient } from "https://esm.sh/v135/@supabase/supabase-js@2.7.0";
+import { ResponseRisk3 } from "deco-sites/niivu-bank/packs/types.ts";
+
+interface defaultFields {
+  phone: string;
+  address: {
+    zip_code: string;
+    street: string;
+    number: string;
+    complement: string;
+    city: string;
+    state: string;
+  };
+  email: string;
+}
+
+export interface CPF extends defaultFields {
+  type: "CPF";
+  rg: string;
+  full_name: string;
+  cpf: string;
+}
+
+export interface CNPJ extends defaultFields {
+  type: "CNPJ";
+  business_name: string;
+  cnpj: string;
+}
+
+async function loginRisk3(
+  usernameSecret: Secret,
+  passwordSecret: Secret,
+  url: string,
+): Promise<ResponseRisk3> {
+  const username = typeof usernameSecret === "string"
+    ? usernameSecret
+    : usernameSecret?.get();
+  const password = typeof passwordSecret === "string"
+    ? passwordSecret
+    : passwordSecret?.get();
+
+  console.log(username, password, url);
+  const res = await fetch(
+    `${url}/api/v0/login?username=${username}&password=${password}`,
+    {
+      method: "POST",
+    },
+  );
+
+  return res.json();
+}
+
+function logoutRisk3(url: string) {
+  return fetch(`${url}/api/v0/logout`, {
+    method: "POST",
+  });
+}
+
+async function analiseCPF(url: string, authToken: string, cpf: string): Promise<ResponseRisk3> {
```

Comment:
Evita misturar ingles/portugues no codigo
```suggestion
async function checkCPF(url: string, authToken: string, cpf: string): Promise<ResponseRisk3> {
```